### PR TITLE
feat: roll to new folio, rewrite to use environments

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,66 +47,77 @@ it("is a basic test with the page", async ({ page }) => {
 The test runner provides browser primitives as arguments to your test functions. Test functions can use one or more of these arguments.
 
 - `page`: Instance of [Page][page]. Each test gets a new isolated page to run the test.
-- `context`: Instance of [BrowserContext][browser-context]. Each test gets a new isolated context to run the test. The `page` object belongs to this context.
-  - `contextOptions`: Default options passed to context creation. Learn [how to modify them](#modify-context-options).
-- `browser`: Instance of [Browser][browser]. Browsers are shared across tests to optimize resources. Each worker process gets a browser instance.
-  - `browserOptions`: Default options passed to browser/launch creation.
+- `context`: Instance of [BrowserContext][browser-context]. Each test gets a new isolated context to run the test. The `page` object belongs to this context. Learn [how to configure](#modify-options) context creation.
+- `browser`: Instance of [Browser][browser]. Browsers are shared across tests to optimize resources. Each worker process gets a browser instance. Learn [how to configure](#modify-options) browser launch.
```
Comment (pavelfeldman) [inline README.md:51]: "Each worker process gets a browser instance." jumps on me with complexity. Let's move this to a dedicated section.

File: README.md [hunk 2]
```diff
@@ -46,66 +47,77 @@ it("is a basic test with the page", async ({ page }) => {
...
+test.describe("feature foo", () => {
+  test("is working correctly", async ({ page }) => {
     // Test function
   });
 });
 ```
 
+### Write a configuration file
+
+Create `config.ts` to configure your tests. Here is an example configuration that runs every test in three browsers.
```
Comment (pavelfeldman) [inline README.md:71]: expand "three" to "Chromium, Firefox and WebKit"

File: README.md [hunk 3]
```diff
@@ -46,66 +47,77 @@ it("is a basic test with the page", async ({ page }) => {
...
-describe("feature foo", () => {
-  it("is working correctly", async ({ page }) => {
+test.describe("feature foo", () => {
+  test("is working correctly", async ({ page }) => {
     // Test function
   });
 });
 ```
 
+### Write a configuration file
```
Comment (pavelfeldman) [inline README.md:69]: Let's get ourselves `npx playwright-test init` instead and move it up to where you create first test file.

**Comment by badsyntax** [comment]
This seems like a breaking change to me, yet it's published under a new minor version. If not semver, what versioning strategy are you using?

**Comment by dgozman** [comment]
@badsyntax This project is still below version 1.0 and is not considered stable. This is what semver has to say about zero majors: https://semver.org/#spec-item-4.

**Comment by badsyntax** [comment]
Thanks. The README makes this obvious but I had just completely missed that this package is in preview. Sorry for the noise :(

**Pull request merged**